_data/meltano/utilities/airflow/apache.yml

```diff
@@ -0,0 +1,91 @@
+name: airflow
+namespace: airflow
+label: Airflow
+docs: https://docs.meltano.com/guide/orchestration
+repo: https://github.com/apache/airflow
+ext_repo: https://github.com/meltano/airflow-ext
+pip_url: git+https://github.com/meltano/airflow-ext.git@main apache-airflow==2.3.3 --constraint https://raw.githubusercontent.com/apache/airflow/constraints-2.3.3/constraints-no-providers-3.8.txt
+executable: airflow_invoker
+variant: apache
+maintenance_status: active
+definition: is an [orchestrator](https://docs.meltano.com/concepts/plugins#orchestrators) that allows for workflows to be programmatically authored, scheduled, and monitored.
+commands:
+  describe:
+    executable: airflow_extension
+    args: describe
+  initialize:
```

Comment:
@pandemicsyn @tayloramurphy the initialize command functionality is slightly confusing for me on first glance because it seems like its mixing both initializing the db and the meltano project assets. The project assets only need to initialized once then git versioned but the db stuff needs to be done in each environment, right?

Maybe all we need is an extra blurb that describes what the initialize command does and when you should run it (i.e. once per project or once per environment). That would take some of the mystery out of it.

Reply:
I think this is way @aaronsteers mentioned adding discrete init and setup in a follow up version: https://github.com/meltano/edk/discussions/23

I can definitely clarify what's going to happen in the docs.  Could also add a dedicated `initdb`, `setup`, `etc`  command (be an easy change).

Reply:
@pnadolny13 fyi forgot that airflow extension works the same as the non-extension version. The db and config get created "automatically" _everytime_ you invoke airflow ...if they don't already exist that is.

It's actually just the dag generator and gitignore that gets created when you call initialize (basically the files-airflow replacement).  If `initialize` is the first thing you call then all three get created at once. But if you go straight from installing airflow to adding a user - you won't deploy a dag generator. I'll update the docs to clarify.

Reply:
I updated the description on the commands - @pandemicsyn will you review?